src/config/config.ts

```diff
@@ -14,6 +14,10 @@ const config = {
       useUnifiedTopology: true,
     },
   },
+  jwt: {
```

Comment:
Would be good to update `env.example` accordingly and apply default values here when possible.